### ISSUE TITLE
US: How does Offline Usage Work?

### ISSUE THREAD

**Issue by iMonZ** [eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework#95]
### Description
Offline access is very important.
Imagine the Internet is not working any more, or you are in a tunnel or underground without Wi-Fi.
Is EUDI going to support Offline access of wallet entries. And if yes, how?

**Comment by piotrblasiak** [comment]
I agree - to some degree using the wallet in offline mode should be possible. For example, being able to verify the name & age of a person by scanning their wallet QR code. Of course, verifying online to make sure the ID has not been withdrawn should be possible but it should not be required to verify online.

**Comment by RokeJulianLockhart** [comment]
Surely cryptography should render this feasible to implement? I can imagine that the issuing authority produces public keys for each saved credential.

**Comment by digeorgi** [comment]
Thank you for your comment. 
The **offline use** is covered by the proximity presentation of attestations under ARF section 4.2.1 "The Presentation Interface". No update to the ARF is considered.
To provide clarity: Proximity flows complying with ISO/IEC 18013-5 use short-range communication channels: QR code or NFC for setting up the communication and BLE, NFC, or Wi-Fi Aware for data transfer. For such flows, it does not matter whether or not the Wallet Instance or the Relying Party Instance have an internet connection."
